store/init_test.go

```diff
@@ -11,7 +11,6 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/celestiaorg/go-header/headertest"
-	"github.com/celestiaorg/go-header/local"
 )
 
 func TestInitStore_NoReinit(t *testing.T) {
```

Comment:
Do we even need this test? Or should we move it to Store

Reply:
But we def need a test case:

NewStore
Start
GetByHeight or Head/Tail -> ErrEmptyStore
Append
GetByHeightt/Head/Tail -> 👍🏻 
Stop
GetByHeight or Head/Tail -> ErrEmptyStore
Start
GetByHeightt/Head/Tail -> 👍🏻

Reply:
Done (moved).